### PR TITLE
feat: first version of rules

### PR DESCRIPTION
**Pull request by yyewolf**

### CODE REVIEW

File: pages/rules.vue [hunk 1]
```diff
@@ -7,8 +7,54 @@
...
+			<p>Welcome to the rules page for our event !</p>
+
+			<p>To ensure a fair and enjoyable experience for everyone, please take a moment to read through the competition guidelines.</p>
+
+			<h1> Team </h1>
+			<ul>
+				<li> Each team is limited to a maximum of 4 members. </li>
+				<li> All members must register an account to be eligible for prizes. </li>
+				<li> Each participant can only be part of one team throughout the event. </li>
+				<li> Collaboration with other teams or external parties is strictly prohibited. </li>
```
Comment (yyewolf) [inline pages/rules.vue]: `external parties` => `third parties`

File: pages/rules.vue [hunk 2]
```diff
@@ -7,8 +7,54 @@
...
+				<li> Teams must choose appropriate names. Offensive or inappropriate names are not allowed. </li>
```
Comment (yyewolf) [inline pages/rules.vue]: `offensive or inappropriate names will be renamed or deleted`

File: pages/rules.vue [hunk 3]
```diff
@@ -7,8 +7,54 @@
...
+				<li> Teams will accumulate points as a group, and individual performance will not affect the team's overall score. </li>
```
Comment (yyewolf) [inline pages/rules.vue]: pas de `and` après la virgule

File: pages/rules.vue (hunk 1 above)
Comment (Nounoursdestavernes) [reply pages/rules.vue]: done

File: pages/rules.vue (hunk 2 above)
Comment (Nounoursdestavernes) [reply pages/rules.vue]: done

File: pages/rules.vue (hunk 3 above)
Comment (Nounoursdestavernes) [reply pages/rules.vue]: done

**Pull request merged**